3D/script_slurm

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#SBATCH --time=48:00:00
+#SBATCH --job-name=3d-unet-horovod
+#SBATCH -o /home/loic.mcgill/logs/%x_%j.out # %x job name, %N for node name, %j for jobID
+#SBATCH --partition=6252
+#SBATCH --nodes=10
+
+data_path=/mnt/lustrefs/mcgill/decathlon/Task01_BrainTumour/
```

Comment:
Probably just need the data_path and script path to be generic instead of the McGill-specific path.